### ISSUE TITLE
Training time

### ISSUE THREAD

**Issue by ariel415el** [VITA-Group/SinNeRF#3]
Hi, 
Thanks for the great work and for uplaoding the training code.

Can you share the time it took to train the model **room.ckpt** and the device it was trained on so that I can have a reference for reproducing/ training for other scenes?

Thanks

**Comment by Ir1d** [comment]
Hi, I don't remember the exact time for training that model. But for all experiments, I train for at most 5000 epochs on at most 4 A6000 GPUs and they take less than a day. For LLFF/room the time to converge is much less and it takes about 3000 epochs to obtain pretty good results. However, also note that NeRF training is sensitive to the initialization. So if you observe a very large loss after several epochs (e.g. depth loss > 10), you might want to restart the training.
